xrayvision/tests/test_clean.py

```diff
@@ -129,14 +129,14 @@ def test_clean_sim():
     sub_uv = np.hstack([sub_uv, np.zeros((2, 1))]) / u.arcsec
 
     # Factor of 9 is compensate for the factor of  3 * 3 increase in size
-    dirty_beam = idft_map(np.ones(321) / 321, u=sub_uv[0, :], v=sub_uv[1, :], shape=(n * 3 + 1, m * 3 + 1))
+    dirty_beam = idft_map(np.ones(321) / 321, u=sub_uv[0, :], v=sub_uv[1, :], shape=(n * 3 + 1, m * 3 + 1) * u.pix)
 
     vis = dft_map(data, u=sub_uv[0, :], v=sub_uv[1, :])
 
-    dirty_map = idft_map(vis, u=sub_uv[0, :], v=sub_uv[1, :], shape=(n, m))
+    dirty_map = idft_map(vis, u=sub_uv[0, :], v=sub_uv[1, :], shape=(n, m) * u.pix)
 
     clean_map, model, res = clean(
-        dirty_map, dirty_beam, clean_beam_width=0.1 * u.arcsec, pixel=[2, 2] * u.arcsec, niter=500
+        dirty_map, dirty_beam, pixel_size=[2, 2] * u.arcsec, clean_beam_width=0.1 * u.arcsec, niter=500
```

Comment:
shouldn't it be `pixel_size=[2, 2] * u.arcsec / u.pix`?

Reply:
Yea I'll fix this.

Reply:
Thanks!

Reply:
Ok so I haven't yet gone through the clean module yet so the codes needs to be updated, same for the men module.